pkg/api/validation/schema.go

```diff
@@ -66,6 +66,19 @@ func NewSwaggerSchemaFromBytes(data []byte) (Schema, error) {
 	return schema, nil
 }
 
+// unpackObject returns obj's items element as an array of interface.
+// Returns an error if obj is not a List type (does not have an items member).
+func unpackObject(obj map[string]interface{}) ([]interface{}, error) {
+	items := reflect.ValueOf(obj).MapIndex(reflect.ValueOf("items"))
+	if !items.IsValid() {
+		return nil, fmt.Errorf("no items field in %#v", obj)
+	}
+	if _, ok := items.Interface().([]interface{}); !ok {
+		return nil, fmt.Errorf("items isn't a slice")
+	}
+	return items.Interface().([]interface{}), nil
```

Comment:
You're expecting `[]map[string]interface{}`, right?  Any reason not to make the signature match that?